al5d_gazebo/scripts/arm_controller.py

```diff
@@ -11,14 +11,17 @@
 from copy import deepcopy
```

Comment:
Have you confirmed that these changes to the arm_controller dont break anything with the previous labs use cases?

Reply:
I tested all the previous labs and I wasn't able to produce any breakages